topic-operator/src/main/java/io/strimzi/operator/topic/Config.java

```diff
@@ -159,6 +165,19 @@ private Value(String key, Type<? extends T> type, boolean required) {
     /** The endpoint identification algorithm used by clients to validate server host name. The default value is https. Clients including client connections created by the broker for inter-broker communication verify that the broker host name matches the host name in the broker’s certificate. Disable server host name verification by setting to an empty string.**/
     public static final Value<String> TLS_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM = new Value<>(TC_TLS_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM, STRING, "HTTPS");
 
+    /**
+     * The store topic for the Kafka Streams based TopicStore
+     */
+    public static final Value<String> STORE_TOPIC = new Value<>(TC_STORE_TOPIC, STRING, "store-topic");
+    /** The store name for the Kafka Streams based TopicStore */
+    public static final Value<String> STORE_NAME = new Value<>(TC_STORE_NAME, STRING, "topic-store");
```

Comment:
This is the name of the store within Kafka Streams?

Reply:
Yes, this is there those Topic instances (from TopicStore) are stored.